tests/run.sh

```diff
@@ -27,3 +28,21 @@ wait_for_ingress_ready 'knative-ingressgateway' 'istio-system'
 kubectl create namespace $NAMESPACE
 fats_create_push_credentials $NAMESPACE
 riff namespace init $NAMESPACE $NAMESPACE_INIT_FLAGS
+
+# run test functions
+echo "Run functions"
+source `dirname "${BASH_SOURCE[0]}"`/../functions/helpers.sh
+
+# uppercase
+for test in java java-boot java-local node npm command; do
```

Comment:
lines 37-45 could be in a file in like `uppercase/run.sh`, WDYT?

Reply:
That would solve https://github.com/projectriff/fats/issues/16 imo.

Reply:
This is what we used to do, and I want to move away from it. FATS provides a library of function that an integrator can choose to invoke. This script is for testing FATS itself.

Reply:
after an offline chat, I got it as well